cobra/core/metabolite.py

```diff
@@ -217,4 +217,4 @@ def summary(self, threshold=0.01, fva=False, floatfmt='.3g', **kwargs):
             return metabolite_summary(self, threshold=threshold, fva=fva,
                                       floatfmt=floatfmt, **kwargs)
         except ImportError:
-            warn('Summary methods require pandas/tabulate')
+            warn('Summary methods require tabulate')
```

Comment:
I think we could just add tabulate to the dependencies as well. It's a pure Python dep and summsry methods are pretty essential by now.